dash/frontend/src/app/modules/private/pages/cluster/kube-bench/kube-bench-report-details/kube-bench-report-details.component.html

```diff
@@ -1,83 +1,101 @@
 <div class="page-container">
   <div class="page-container-content">
-    <div class="page-title-button-group-spacing">
-      <div class="page-title">
-        Kube Bench Test Results
-      </div>
-      <div *ngIf="report && report.Totals">
-        <div class="page-title">
-          {{report.Totals.total_pass}}/{{report.Totals.total_pass + report.Totals.total_fail + report.Totals.total_warn}} Tests Passed
-        </div>
-      </div>
+    <div class="page-title">
+      Kube Bench Test Results
+    </div>
+    <div *ngIf="report && report.Totals" style="font-weight: bold; margin-bottom: 1em;">
```

Comment:
Styles should be applied via classes rather than inline styles where possible.
For bolding, you could also just use a <strong> tag.

Reply:
Good catch! I was playing inline to try to get things to work and forgot to clean up this line.